docs/Custom-Brimcap-Config.md

```diff
@@ -53,12 +52,17 @@ additional customizations.
 
 1. The Brimcap-bundled Zeek includes the additional packages
 [geoip-conn](https://github.com/brimdata/geoip-conn),
-[zeek-community-id](https://github.com/corelight/zeek-community-id),
 [HASSH](https://github.com/salesforce/hassh),
 and [JA3](https://github.com/salesforce/ja3). These would typically be
 installed via [Zeek Package Manager](https://docs.zeek.org/projects/package-manager/en/stable/quickstart.html).
 
-2. The Brimcap-bundled Suricata includes a
+2. Other changes are made to the default configuration of the Brimcap-bundled
+Zeek, such as enabling
+[Community ID Flow Hashing](https://docs.zeek.org/en/master/customizations.html#community-id).
+See the [build-zeek release automation](https://github.com/brimdata/build-zeek/blob/main/.github/workflows/release.yml)
+for details on how this and other customizations are handled.
```

Comment:
We used to use a Zeek package for Community ID, but now it's included out-of-the-box with Zeek so I've been taking advantage of that in the Zeek builds that come out of the new build-zeek repo. This gave me a nice excuse to link to the build-zeek repo, since it's a better starting place than what we had before for users that want to try their hand at making their own custom Zeek builds, including on Windows.